extism-maturin/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "maturin"
 
 [project]
 name = "extism-sys"
-version = "0.0.0-replaced-by-ci"
+version = "0.0.0+replaced-by-ci"
```

Comment:
This makes the placeholder version string compatible with pyproject. Maturin does not accept the hyphenated version; in order to develop locally one would have to replace the version out and remember not to commit the change.